docs/changelogs/v0.18.md

```diff
@@ -68,6 +69,24 @@ $ curl "http://127.0.0.1:8080/ipfs/$DIR_CID?format=dag-json" | jq
 }
 ```
 
+#### Automated default `Routing.Type`
+
+The default `Routing.Type` is changed from `dht` to `auto`.
+
+When `Routing.Type` is unset or set to `auto`, your node will accelerate some types of routing
+by leveraging HTTP endpoints compatible with [IPIP-337](https://github.com/ipfs/specs/pull/337)
+in addition to the IPFS DHT. DHT response timeout in `auto` mode is 5 minutes, and IPNI is 3 seconds.
+By default, an instance of IPNI ([InterPlanetary Network Indexer](https://github.com/ipni/specs/blob/main/IPNI.md#readme))
+at https://cid.contact is used.
+
```

Comment:
```suggestion

By integrating with IPNIs like cid.contact, it's possible to find CIDs that are stored by Filecoin Storage Providers (SPs) and potentially retrieve it if the SPs service Bitswap requests.  As of this release, some SPs are advertising Bitswap.  You can follow the roadmap progress for IPNIs and Bitswap in SPs [here](https://github.com/ipfs/kubo/pull/9475) 
```

@dvd @willscott (and Lauren although I don't know here github): If there's a better link to point people at to learn more about this and follow the progress, feel free to share.

Reply:
@LaurenSpiegel is Lauren :)

https://github.com/orgs/filecoin-project/projects/29/views/1 or https://pl-strflt.notion.site/Retrieval-Milestones-5ea2405b34524ab68af50bd7e089190f are probably the right places to track roadmap

Reply:
Doh - so I meant to link to https://www.starmaps.app/roadmap/github.com/protocol/bedrock/issues/1 and not https://github.com/ipfs/kubo/pull/9475.

I was thinking to go the starmaps route because there is going to be org pressure to keep that updated.

https://github.com/orgs/filecoin-project/projects/29/views/1 is lower level and boost focused
https://pl-strflt.notion.site/Retrieval-Milestones-5ea2405b34524ab68af50bd7e089190f has many things from the past so doesn't give the sense of being fresh.

Reply:
Incorporated in https://github.com/ipfs/kubo/pull/9475/commits/903fc25406b72850396862ba5061da075c5d1cfc